content/docs/use-cases/gitops-for-kubeflow.md

```diff
@@ -1,16 +1,18 @@
-+++
```

Comment:
The three plus signs `+++` must remain in place. Their removal is causing incorrect rendering at the top of the page. See preview.

Reply:
Sorry, forgot to add those while committing. Added those in the new commits.